waspc/cli/src/Wasp/Cli/Question.hs

```diff
@@ -0,0 +1,86 @@
+module Wasp.Cli.Question
+  ( query,
+    queryWithOptions,
+    queryNotNull,
+  )
+where
+
+import Data.List (intercalate)
+import qualified Data.Text as T
+import System.IO (hFlush, stdout)
+import Text.Read (readMaybe)
+import qualified Wasp.Util.Terminal as Term
+
+queryNotNull :: String -> IO String
+queryNotNull question = do
+  ensureNotNull . query $ question
+
+queryWithOptions :: String -> [String] -> IO (Maybe String)
+queryWithOptions question options = do
+  putStrLn optionsStr
```

Comment:
When I am reading this function, seeing just `optionsStr` makes it very unclear for me as to what this is.